common/app/common/LinkTo.scala

```diff
@@ -98,6 +98,7 @@ class CanonicalLink {
     "index",
     "page",
     "filterKeyEvents",
+    "iframe_parent_component",
```

Comment:
Minor thing, but it appears the convention here is camelCasing rather than snake_casing